### ISSUE TITLE
Tutorial prominently mentions weight decay, but does not implement it

### ISSUE THREAD

**Issue by samuela** [google-deepmind/optax#308]
[This heading in optax-101.ipynb](https://github.com/deepmind/optax/blob/7c76078015469ad00805063cfd772c1564f37640/docs/optax-101.ipynb?short_path=6fc9dea#L194) is misleading since weight decay is not actually used anywhere in the example AFAICT.

**Comment by rosshemsley** [comment]
I think the "weight decay" here is referring to the use of `adamw`:

https://optax.readthedocs.io/en/latest/api.html#adamw

instead of adam

Perhaps it's a bit subtle indeed though! - Would you prefer to see an explicit example of adding a weight decay term in the chain, or would it be enough to highlight this more?

**Comment by samuela** [comment]
Ahhh I see. I was looking for something like
```
tx = optax.chain(optax.add_decayed_weights(5e-4), optax.sgd(lr_schedule, momentum=0.9))
```

> Perhaps it's a bit subtle indeed though! - Would you prefer to see an explicit example of adding a weight decay term in the chain, or would it be enough to highlight this more?

Personally speaking I like the idea of making it explicit, if for no other reason than that it lets you showcase another feature of optax, `opted.chain`.

**Comment by jankolf** [comment]
Hi,
is there something similar to the example of @samuela in optax, as I need to use weight decay together with SGD?
Perhaps using the optax.chain style as mentioned above?
In [Pytorch there is a parameter to add weight decay to SGD](https://pytorch.org/docs/stable/generated/torch.optim.SGD.html) and I am currently looking for something similar.

Thanks in advance.
